conf/pipeline/pipeline.yaml

```diff
@@ -24,11 +24,19 @@ etcd:
     cert_key_file: "${ETCD_CERT_KEY_FILE:/certs/etcd-client-key.pem}"
     ca_file: "${ETCD_CA_FILE:/certs/etcd-ca.pem}"
 
-etcd-election@reconciler:
-  root_path: erda/component-leader/pipeline/reconciler
+pipengine:
```

Comment:
I think

`pipengine: {}`

is more better, because it is disambiguation.

Reply:
it make sense.